maintainers/maintainer-list.nix

```diff
@@ -5960,6 +5960,12 @@
     githubId = 25820499;
     name = "Roman Kretschmer";
   };
+  goatchurch = {
+    email = "julian@goatchurch.org.uk";
+    github = "goatchurchprime";
+    githubId = 677254;
+    name = "Julian Todd";
+  };
```

Comment:
As @MatthewCroughan already said, this should be it's own commit called `maintainers: add goatchurch` otherwise this looks good to me.